docs/ldml/tr35-keyboards.md

```diff
@@ -666,6 +666,8 @@ _Attribute:_ `longPressKeyIds="{list of key ids}"` (optional)
 >
 > In a list of keys specified by `longPressKeyIds`, the key matching `longPressDefaultKeyId` attribute (if present) specifies the default long-press target, which could be different than the first element. It is an error if the `longPressDefaultKeyId` key is not in the `longPressKeyIds` list.
 >
+> Gestures are not nested: other gestures (such as flick, multiTap, longPress) are ignored on the keys in the `longPressKeyIds` list.
```

Comment:
Like before, do not use "are" and similar factual statements. It is unclear who they target and what they mean. Say "Implementations may ignore/not support" or shall not or whatever is appropriate.

Reply:
Well put. I will revise.

Reply:
updated